src/SmartFormat/SmartFormatter.cs

```diff
@@ -166,8 +191,20 @@ public string Format(IFormatProvider? provider, string format, params object[] a
         /// <param name="args">The objects to format.</param>
         public void FormatInto(IOutput output, string format, params object[] args)
         {
+            FormatInto(output, args, format);
+        }
+
+        /// <summary>
+        /// Writes the formatting result into an <see cref="IOutput"/> instance.
+        /// </summary>
+        /// <param name="output">The <see cref="IOutput"/> where the result is written to.</param>
+        /// <param name="args">The objects to format.</param>
+        /// <param name="format">The format string.</param>
+        public void FormatInto(IOutput output, IList<object> args, string format)
+        {
+            args = args ?? k_Empty;
```

Comment:
```suggestion
            args ??= k_Empty;
```
`args`is not marked `nullable`. In this case, the line is redundant.